### DISCUSSION TITLE
Rose Pine theme for Intellij

### DISCUSSION THREAD

**Discussion by ridiculouswaffle** [rose-pine/rose-pine-theme#105] [Ideas]
It would be nice if there is a theme for Intellij Idea. And yes I checked [this](https://github.com/Jmorjsm/rose-pine-intellij) and I am not satisfied with it.

There are a lot of flaws with the current rose pine theme.

I can help you in developing the theme. I just want an official & proper theme with no default Darcula colours.

**Comment by ThatOneCalculator** [comment]
You could just contribute to the https://github.com/Jmorjsm/rose-pine-intellij repository :)
